functions/lib/stripe/webhooks.py

```
@@ -718,6 +740,17 @@ def stripe_webhook_checkout_fulfilment(req: https_fn.Request) -> https_fn.Respon
 
             line_items = session.line_items
             customer_details = session.customer_details
+            payment_intent_id = session.payment_intent
+            try:
+                capture_method = stripe.PaymentIntent.retrieve(
+                    payment_intent_id,
+                    stripe_account=connected_account_id,
+                ).capture_method
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)